src/components/Tables/Table.tsx

```diff
@@ -618,6 +661,19 @@ const TableStyling = styled.div`
     & .rdt_TableRow:nth-child(odd) {
         background-color: ${(props) => props.theme.main.background[0]};
     }
+
+    ${(props) =>
+        props.rowBreakPointConfigs
+            .map((breakPointConfig) => {
+                return breakPointConfig.breakPoints
+                    .map(
+                        (breakPoint) => `& .rdt_TableRow:nth-child(${breakPoint + 1}) {
+                            border-top: ${props.dividingLineStyleOptions[breakPointConfig.dividingLineStyle].thickness} solid ${props.dividingLineStyleOptions[breakPointConfig.dividingLineStyle].colour};
+                        }`
+                    )
+                    .join();
+            })
+            .join()}
```

Comment:
This is quite a lengthy bit, might be worth adding a comment about what this does and how

Reply:
done